xml/ha_migration.xml

```diff
@@ -748,7 +748,7 @@
       Restart the cluster stack on the upgraded node to make the node rejoin
       the cluster:
      </para>
-<screen>&prompt.root;<command>systemctl</command> start pacemaker</screen>
+<screen>&prompt.root;<command>crm</command> cluster restart</screen>
```

Comment:
Although it says "Restart the cluster stack ..." in here, but it's based on the "stop" in the first step. So "crm cluster start" would be good enough in here:-)

Reply:
@gao-yan : Thanks, fixed with fixed with https://github.com/SUSE/doc-sleha/commit/79492f55.